### ISSUE TITLE
Plot Template Matching Example throws ImportError: cannot import name 'template_matching' from 'maad.rois' 

### ISSUE THREAD

**Issue by Bengt** [scikit-maad/scikit-maad#63]
```bash
Traceback (most recent call last):
  File "/opt/project/example_gallery/1_basic/plot_template_matching_example.py", line 18, in <module>
    from maad.rois import template_matching
ImportError: cannot import name 'template_matching' from 'maad.rois' (/opt/projects/cst.maad-dash/venv/lib/python3.10/site-packages/maad/rois/__init__.py)
```

**Comment by juansulloa** [comment]
Hi Brengt,

Template matching is currently under development on the 'production' branch. 
How did you get this error?

**Comment by Bengt** [comment]
Hi Juan,

I checked out the default branch, which is the production branch. From the error message I take that I installed scikit-maad system-wide as I was instructed to in the README. Then, I ran the example installed with the package into my OS.

Regards,
Bengt

**Comment by juansulloa** [comment]
Hi Bengt,

The template matching function was included on the new 1.4.0 version of scikit-maad, so you should not have any issues running the [template matching example](https://scikit-maad.github.io/_auto_examples/1_basic/plot_template_matching_example.html#sphx-glr-auto-examples-1-basic-plot-template-matching-example-py).

All the best,
